types/setting.go

```diff
@@ -1592,6 +1595,16 @@ var (
 		ReadOnly: false,
 		Default:  "false",
 	}
+
+	SettingDefinitionMaxPodRecreateBackoff = SettingDefinition{
+		DisplayName: "Max Pod Recreate Backoff",
+		Description: "Maximum backoff duration (in seconds) between pod recreation attempts. The interval doubles after each failure, up to this limit. Default is 120 seconds.",
```

Comment:
Hello @sushant-suse (doc engineer), could you review the `DisplayName` and `Description`? Thank you.